### PR TITLE
Replaces the ARC interior & ARC tweaks

### PR DESCRIPTION
**Pull request by Zonespace27**
# About the pull request

- Replaces the ARC interior with a better-looking one, sprite made by Zydras
- Modifies the ARC model to only have one door.
- ARC now fits 5 instead of 7 (including reserved slots)
- ARC now fits 4 xenoes intead of 5  
- Fixed vehicle elevator causing vehicle sprites to be incorrectly offset/directional 
- RE700 armor pen 10->15

# Explain why it's good for the game

- It looks cooler and makes the ARC feel less like a TARDIS
- Necessary for the new ARC model, plus Lore Accuracy(tm)
- It wasn't meant to fit 7 in the first place
- There's a total of 5 walkable tiles in the ARC and xenomorphs are a fair bit larger than humans, so I considered this fair
- Longstanding bug fixed. Fixes https://github.com/cmss13-devs/cmss13/issues/6362
- The gun's working as an okay deterrent against t1s and other unarmored xenoes, but it isn't enough of a deterrent to the more heavily-armored ones.

# Testing Photographs and Procedure

<details>

image

</details>

# Changelog
:cl: Zonespace, Zydras
fix: Vehicle elevator no longer incorrectly rotates vehicles
add: New ARC exterior and interior
balance: ARC now fits 5 humans instead of 7
balance: ARC now fits 4 xenos intead of 5
balance: ARC's RE700 turret armor pen 10->15
balance: On account of the smaller size, the ARC no longer has a nanomed.
/:cl:

### CODE REVIEW

File: code/modules/vehicles/arc/arc.dm [hunk 1]
```diff
@@ -17,11 +17,10 @@
...
 	interior_map = /datum/map_template/interior/arc
 
-	passengers_slots = 3
-	xenos_slots = 5
+	passengers_slots = 2 // 5 total. Reserved slots are added to passenger slots.
+	xenos_slots = 4
 
 	entrances = list(
-		"left" = list(2, 0),
 		"right" = list(-2, 0),
```
Comment (Drulikar) [inline code/modules/vehicles/arc/arc.dm:24]: Is it possible to make two tiles valid spaces to enter the vehicle? Visually this turf should be a good spot to enter the vehicle to me: 
image

File: maps/interiors/arc.dmm [hunk 2]
```diff
@@ -1,296 +1,112 @@
...
-	pixel_y = 8
-	},
-/obj/effect/landmark/interior/spawn/entrance{
-	alpha = 50;
-	dir = 1;
-	exit_type = /obj/structure/interior_exit/vehicle/apc;
-	name = "Left APC door";
-	pixel_y = 32;
-	tag = "left"
+	layer = 2.98;
```
Comment (Drulikar) [inline maps/interiors/arc.dmm:69]: Layering looks to be incorrect: 
image

**Review by Drulikar — CHANGES_REQUESTED** [review]
Otherwise looks good and appears to be working.

File: maps/interiors/arc.dmm (hunk 2 above)
Comment (Drulikar) [reply maps/interiors/arc.dmm:69]: Also slightly funny to me 3 of the 4 computers to the left all can open tacmap but its fine.

File: code/modules/vehicles/arc/arc.dm (hunk 1 above)
Comment (Zonespace27) [reply code/modules/vehicles/arc/arc.dm:24]: Doesn't seem to be doable without reworking a fair bit of entrance code unfortunately

File: maps/interiors/arc.dmm (hunk 2 above)
Comment (Zonespace27) [reply maps/interiors/arc.dmm:69]: fixed

Comment (Drulikar) [reply maps/interiors/arc.dmm:69]: Still appears to be an issue:
image

Comment (Zonespace27) [reply maps/interiors/arc.dmm:69]: Fixed for real this time

**Review by Drulikar — APPROVED** [review]

**Review by nauticall — APPROVED** [review]
sprites 👍

**Pull request merged**